src/pages/CreateClub/CreateClub.tsx

```diff
@@ -81,16 +82,64 @@ export const CreateClub: React.FC = () => {
               onChange={onChange}
               value={formData.name}
             />
-
+          </>
+        );
+      case 2:
+        return (
+          <>
+            <p className={style.description}>
+              Give your club a plot twist! Share what makes it unique and why
+              others will want to join.
+            </p>
             <FormInput
-              type="text"
+              type="textarea"
               name="description"
               placeholder="Club Description"
               required
               onChange={onChange}
               value={formData.description}
+              rows={4}
             />
-          </FormWrapper>
+          </>
+        );
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <Layout>
+      <div className={style.createClubPage}>
+        {isSubmitted ? (
+          <div>
+            <h1>Club created successfully!</h1>
+            <p className={style.description}>
+              view your brand new{' '}
+              <Link to={`/clubs/${id}`} className={style.link}>
+                {formData.name}
+              </Link>
+            </p>
+          </div>
+        ) : (
+          <div className={style.formSection}>
+            <FormWrapper
+              title="Create Club"
+              onSubmit={onSubmit}
+              submitText={currentStep === 1 ? 'Next' : 'Create'}
+            >
+              <div className={style.stepIndicator}>Step {currentStep}/2</div>
+              {renderStep()}
```

Comment:
Even this approach is fine. What do you think, we move this to a component? 

Or perhaps we can just do two SIMPLE conditional renderings, right here. Maybe most renderStep and component are overkill for two steps with two components each